pxr/base/lib/js/CMakeLists.txt

```diff
@@ -19,6 +19,9 @@ pxr_library(js
         types.h
 )
 
+# This removes warnings generated by rapidjson
+target_compile_definitions(js PRIVATE RAPIDJSON_HAS_CXX11_RVALUE_REFS=1)
```

Comment:
Instead of adding this define here, can we add a #define in js/json.cpp, near where the other RAPIDJSON defines are located? rapidjson includes are contained to this one file, and it'd be good to centralize all of the necessary configuration there.

Reply:
Yeah that sounds fine by me 👍